scripts/registerScript.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+numUsers=$1
+if [ -z "$1" ]
+  then
+    numUsers=0
+fi
+echo "Starting politeiad..."
+tmux new-session -d -s politeiad 'politeiad'
+
+# Wait a few seconds to ensure that politeiawww is ready for requests
+echo "Waiting for politeiad to start..."
+sleep 3
+
+# Start politeiawww in basic piwww mode to create first user that will
+# be the admin user
+echo "Starting politeiawww..."
+tmux new-session -d -s politeiawww 'politeiawww --mode=piwww'
```

Comment:
Instead of requiring that user has tmux and then doing all this starting and stopping, why not not create a dbutil command for adding the first cms admin.

Starting politeiawww in piwww mode to add an admin is really not a good way to do it.